war/gulpfile.js

```diff
@@ -23,7 +23,7 @@ builder.bundle('src/main/js/pluginSetupWizard.js')
     .withExternalModuleMapping('bootstrap', 'core-assets/bootstrap:bootstrap3', {addDefaultCSS: true})
     .withExternalModuleMapping('handlebars', 'core-assets/handlebars:handlebars3')
     .less('src/main/less/pluginSetupWizard.less')
-    .inDir('src/main/webapp/jsbundles');
+    .inDir('target/classes/org/jenkinsci/setup'); // bundle as an adjunct
```

Comment:
@tfennelly I don't understand this change. How will be exposed the files through a HTTP request? Now it is clear. Both `pluginSetupWizard.js` and `pluginSetupWizard.css` are exposed as `[context]/jsbundles/`.